Packs/Synapse/Integrations/Synapse/Synapse.py

```diff
@@ -547,34 +523,26 @@ def url_reputation_command(client, args, good_tag, bad_tag):
             dbot_score=dbot_score
         )
 
-        url_standard_list.append(url_standard_context)
-        url_data_list.append(url_data)
-
-    readable_output = tableToMarkdown('URL List', url_data_list)
-
-    results = CommandResults(
-        readable_output=readable_output,
-        outputs_prefix='Synapse.URL',
-        outputs_key_field='url',
-        outputs=url_data_list,
-        indicators=url_standard_list
-    )
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('URL List', url_data),
```

Comment:
Given a single URL, I feel that URL List is not a good human-readable title, perhaps URL Details?